lib/sensors/temperature.cpp

```diff
@@ -0,0 +1,27 @@
+#include "temperature.hpp"
+
+namespace hyped::sensors {
+
+Temperature::Temperature(hyped::io::I2c &i2c, hyped::core::ILogger &log) : log_(log), i2c_(i2c)
+{
+}
+
+Temperature::~Temperature()
+{
+}
+
+std::optional<uint16_t> Temperature::read()
+{
+  // TODOLater: read from i2c
+  log_.log(hyped::core::LogLevel::kFatal, "Temperature read not implemented");
+  return std::nullopt;
+}
+
+bool Temperature::configure()
+{
+  // TODOLater: configure i2c
```

Comment:
So what does this actually implement? Doing the I/O is the only non-trivial bit.

Reply:
do we have the datasheet?

Reply:
yes

Reply:
then implement it.

Reply:
@HTTYDKing

Reply:
I think that was the intention of this pr anyway? We weren't going to merge this without the temp sensor working

Reply:
`// TODO` not `// TODOLater`